src/Jobs/QueueExportClass.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Maatwebsite\Excel\Jobs;
+
+use Throwable;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class QueueExportClass implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * @var object
+     */
+    public $export;
+
+    /**
+     * @var array
+     */
+    public $filePath;
+
+    /**
+     * @var string|null
+     */
+    public $disk;
+
+    /**
+     * @var string|null
+     */
+    public $writerType;
+
+    /**
+     * @var array
+     */
+    public $diskOptions;
+
+    /**
+     * @var int
+     */
+    public $tries;
+
+    /**
+     * Create a new job instance.
+     *
+     * @param object $export
+     * @param string $filePath
+     * @param string|null $disk
+     * @param string|null $writerType
+     * @param array $diskOptions
+     * @param int|null $tries
+     */
+    public function __construct($export, string $filePath, string $disk = null, string $writerType = null, $diskOptions = [], $tries = null)
+    {
+        $this->export      = $export;
+        $this->filePath    = $filePath;
+        $this->disk        = $disk;
+        $this->writerType  = $writerType;
+        $this->diskOptions = $diskOptions;
+        $this->tries       = $tries;
```

Comment:
I would prefer to get this kind of information via the Export class itself. So if someone declares the tries property on the export class, the queued job should get it from there.